src/components/org/PositionTimeline/SelectableTimeline/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+import { SelectMode, TimelineSplit, TimelinePosition } from '../model';
+import RotationSequence from '../RotationSequence';
+import TimelineProvider from '../TimelineProvider';
+import { ActionSlot } from './ActionSlot';
+import { InfoSlot } from './InfoSlot';
+import { PersonSlot } from './PersonSlot';
+
+type SelectableTimelineProps = {
+    selectedPosition: TimelinePosition;
+    selectMode: SelectMode;
+    initialSplit?: TimelineSplit;
+    initialDate?: Date;
+    previewDates?: boolean;
+    onDateChange?: (date: Date) => void;
+    onSplitSelect?: (split: string) => void;
+    highlighted?: string[];
+    disabled?: string[];
+};
+
+export const SelectableTimeline: FC<SelectableTimelineProps> = ({
+    selectedPosition,
+    selectMode,
+    initialSplit,
+    initialDate,
+    previewDates,
+    onDateChange,
+    onSplitSelect,
+    highlighted,
+    disabled
+}) => {
+    return (
+        <TimelineProvider
+            selectedPosition={selectedPosition}
+            mode={selectMode}
+            initialSplit={initialSplit}
+            initialDate={initialDate}
+            previewDates={previewDates}
+            onDateChange={onDateChange}
+            onSplitSelect={onSplitSelect}
+            highlighted={highlighted}
+            disabled={disabled}
+        >
+            <RotationSequence
+                PersonSlot={PersonSlot}
```

Comment:
Should the user of the component have the opportunity to provide these components them self?

Reply:
Custom slot components may now be provided as props through the wrapper component.